packages/2018-disaster-resilience/src/components/YouAndYourNeighbors/index.js

```diff
@@ -1,25 +1,357 @@
+    shaking: 'Severe',
+    description: 'Damage slight in specially designed structures; considerable damage in ordinary substantial buildings with partial collapse. Damage great in poorly built structures. Fall of chimneys, factory stacks, columns, monuments, walls. Heavy furniture overturned.',
+  },
+  9: {
+    shaking: 'Violent',
+    description: 'Damage considerable in specially designed structures; well-designed frame structures thrown out of plumb. Damage great in substantial buildings, with partial collapse. Buildings shifted off foundations.',
+  },
+  10: {
+    shaking: 'Extreme',
+    description: 'Some well-built wooden structures destroyed; most masonry and frame structures destroyed with foundations. Rails bent.',
+  },
+};
+
+const landslidesScale = {
+  None: {
+    scale: 0,
+    description: 'No potential permanent ground deformation due to landslides',
+  },
+  Low: {
+    scale: 1,
+    description: 'Low potential permanent ground deformation due to landslides (0 - 4 inches)',
+  },
+  Moderate: {
+    scale: 2,
+    description: 'Moderate potential permanent ground deformation due to landslides (4 - 12 inches)',
+  },
+  High: {
+    scale: 3,
+    description: 'High potential permanent ground deformation due to landslides (12 - 39 inches)',
+  },
+  'Very High': {
+    scale: 4,
+    description: 'Very high potential permanent ground deformation due to landslides (39 - 173 inches)',
+  },
+};
+
+const liquefactionScale = {
+  None: {
+    scale: 0,
+    description: 'No potential permanent ground deformation due to liquefaction',
+  },
+  Low: {
+    scale: 1,
+    description: 'Low potential permanent ground deformation due to liquefaction (0 - 4 inches)',
+  },
+  Moderate: {
+    scale: 2,
+    description: 'Moderate potential permanent ground deformation due to liquefaction (4 - 12 inches)',
+  },
+  High: {
+    scale: 3,
+    description: 'High potential permanent ground deformation due to liquefaction (12 - 39 inches)',
+  },
+  'Very High': {
+    scale: 4,
+    description: 'Very high potential permanent ground deformation due to liquefaction (39 - 173 inches)',
+  },
+};
 
 export class YouAndYourNeighbors extends React.Component {
+
   componentDidMount() {
-    // initialize data here
+    this.props.init();
   }
 
   render() {
+    const {
+      isLoading,
+      error,
+      data,
+      isCoordsLoading,
+      coordsError,
+      coordsData,
+      selectedCoords,
+      setCoordinates,
+    } = this.props;
+
+    const geocoderChange = viewport => setCoordinates({ latitude: viewport.latitude, longitude: viewport.longitude });
+    const coordProperties = (coordsData && coordsData.features.length > 0) && coordsData.features[0].properties;
+    const noCoordsData = coordsData && coordsData.features.length < 1;
+
+    const overlay = !data
+    ? null
+    : (
+      <IconMap
+        data={data.features}
+        pickable
+        opacity={0.5}
+        iconAtlas="https://i.imgur.com/xgTAROe.png"
```

Comment:
Can we serve this image ourselves? It'd just be a bummer if it 404'd in the future.